render/persistentvolume.go

```diff
@@ -116,3 +120,82 @@ func (v pvTocontrollerRenderer) Render(rpt report.Report) Nodes {
 	}
 	return Nodes{Nodes: nodes}
 }
+
+// SCToSPCRenderer is a Renderer which produces a renderable SPC Disk
+var SCToSPCRenderer = scTospcRenderer{}
```

Comment:
I think this whole functionality should go into PVToStorageClassrenderer because that is where we are adding the adjacency to storageClasses.